pkg/trcapi/trctest/trc.go

```diff
@@ -60,6 +70,7 @@ func WithHandler(t api.MessageType, h Handler) Option {
 	}
 }
 
+// WithHandler allows to specify a default handler for Conn.
```

Comment:
comment on exported function WithDefaultHandler should be of the form "WithDefaultHandler ..."